src/app/modules/main/wallet_section/send/network_model.nim

```diff
@@ -148,7 +148,13 @@ QtObject:
       self.items[i].amountOut = ""
       self.items[i].resetToNetworks()
       self.items[i].hasGas = true
-      self.dataChanged(index, index, @[ModelRole.AmountIn.int, ModelRole.ToNetworks.int, ModelRole.HasGas.int, ModelRole.AmountOut.int])
+      self.items[i].isEnabled = true
+      self.items[i].isPreferred = true
+      self.items[i].locked =  false
+      self.items[i].lockedAmount = ""
+      self.dataChanged(index, index, @[ModelRole.AmountIn.int, ModelRole.ToNetworks.int, ModelRole.HasGas.int,
+        ModelRole.AmountOut.int, ModelRole.IsEnabled.int, ModelRole.IsPreferred.int, ModelRole.Locked.int,
+        ModelRole.LockedAmount.int])
```

Comment:
Ah sorry for not noticing earlier; if you are indeed resetting everything here, you can now even do the simpler:
```qml
self.dataChanged(index, index)
```